content/ch-applications/vqe-molecules.ipynb

```diff
@@ -166,7 +166,11 @@
     "np.random.seed(999999)\n",
     "target_distr = np.random.rand(2)\n",
     "# We now convert the random vector into a valid probability vector\n",
-    "target_distr /= sum(target_distr)"
+    "target_distr /= sum(target_distr)\n",
+    "\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\", category=DeprecationWarning)\n",
```

Comment:
Can we remove these lines that hide the deprecation warnings please? If the functions are deprecated we should update them (in another PR) instead of ignoring them as they will eventually break the notebook when the functionality is removed in a future release.